### PR TITLE
[UIEUS-379] Adjust permissions and interface versions

### PR DESCRIPTION
**Pull request by alb3rtino**
https://folio-org.atlassian.net/browse/UIEUS-379

https://github.com/folio-org/mod-erm-usage/pull/185 requires adjustments to permissions and ` aggregator-settings` interface version.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -105,6 +105,7 @@
         "subPermissions": [
           "settings.erm-usage.enabled",
           "aggregatorsettings.collection.get",
+          "aggregatorsettings.exportcredentials.item.get",
           "aggregatorsettings.item.get",
```
Comment (elsenhans) [inline package.json:109]: Do we need to keep this permission for older mod-er-usage versions?
Here this permission is deleted among others:
https://github.com/folio-org/mod-erm-usage/pull/185/files

Comment (elsenhans) [reply package.json:109]: Ok, I see, the permissions are not replaced, they are still used.

Comment (alb3rtino) [reply package.json:109]: So no objections?

**Review by elsenhans — APPROVED** [review]
Looking good!

**Pull request merged**